.github/workflows/generated-pipeline.yml

```
@@ -0,0 +1,54 @@
+      with:
+        context: .
+        push: true
+        tags: gcr.io/${{ secrets.GCP_PROJECT_ID }}/my-app:${{ github.sha }}
+
+    - name: Deploy to Cloud Run
+      run: |
+        gcloud run deploy my-app \
+          --image gcr.io/${{ secrets.GCP_PROJECT_ID }}/my-app:${{ github.sha }} \
+          --platform managed \
+          --region ${{ secrets.GCP_REGION }} \
+          --allow-unauthenticated
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)